README.md

```diff
@@ -1,8 +1,12 @@
 # Lunch-Hackers
 
 # Summary
-There are some number of problems each week that the group will try to solve for the next Friday.
-We go through our solutions one by one for the group to explain and hopefully propagate knowledge.
+
+Every two weeks, we will have a few problems that the group will try to solve. After the two weeks are up (technically on a Friday evening), we will schedule a weekend video call at some time that is convenient for everyone who has submitted a solution (hopefully this will settle into a particular time with some regularity).
+
+Each week's problems will be listed in this repo.
+
+To get added to our private list of people for the video call, just open an issue here and we will get you on it.
```

Comment:
Instead of an issue, why not have someone who's interested open a PR to the list of participants? Maybe we could have the list of participants be a part of this file.